GNUmakefile

```diff
@@ -2,7 +2,7 @@ TEST?=$$(go list ./... |grep -v 'vendor')
 GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 WEBSITE_REPO=github.com/hashicorp/terraform-website
 
-MODULE   = $(shell env GO111MODULE=on go list -m)
+MODULE   = $(shell env GO111MODULE=on go list)
```

Comment:
Should we rename this variable to `PACKAGE`  now that we are getting package name instead of module ?

Reply:
Good catch, renamed.